src/Parser/Daedalus.g4

```diff
@@ -63,9 +63,10 @@ varValueDecl: nameNode;
 parameterList: '(' (parameterDecl (',' parameterDecl)*? )? ')';
 parameterDecl: Var typeReference nameNode ('[' arraySize ']')?;
 statementBlock: '{' ( ( (statement ';')  | ( ifBlockStatement ( ';' )? ) ) )*? '}';
-statement: assignment | returnStatement | constDef | varDecl | funcCall;
+statement: assignment | equalityCheck | returnStatement | constDef | varDecl | funcCall | reference | Identifier;
 funcCall: nameNode '(' ( funcArgExpression ( ',' funcArgExpression )*? )? ')';
 assignment: reference assignmentOperator expressionBlock;
+equalityCheck: reference eqOperator expressionBlock;
```

Comment:
We already have eqExpression in expression rule so this has no use.
Also, in the past we allowed our compiler to have values and expressions as statements, so it was possible to have lines like:
```3 > 2;```
or ```5;``` because we found lines like these in original gothic scripts. We however discovered that those lines were bugs in the scripts (or decompiler failed to decompile scripts correctly) and weren't intented to be there.
If we want to support expressions as statements again, then the only we only need to add expression rule in the right place.
As you previously mentioned, having statements like ```3;```matters for mods using Ikarus and for hax but I think it could be a good idea to don't support it and instead have a transpiler that would transpile mods to newer version of Deadalus. Other solution is to make these statements generate warnings or even errors with `--strict` flag.

Reply:
regarding statements like `5;` : if i remember correctly, these are sometimes used to push values on the stack, to be consumed by the calling function (something like C# `out`-parameters i guess, but a lot less maintainable)